### DISCUSSION TITLE
Webstudio AI Alpha is not working

### DISCUSSION THREAD

**Discussion by golocomedia** [webstudio-is/webstudio-community#200] [Q&A]
Webstudio AI Alpha is only showing an error. Is this a persistent problem for others, too?
<img width="1742" alt="Skjermbilde 2024-11-19 kl  18 39 16" src="https://github.com/user-attachments/assets/1c854546-636e-4c47-a722-6a18215a6e81">

**Comment by golocomedia** [comment]
It worked all of a sudden, if I give it simple and clear instructions. I want to add guidelines instructions and color instructions, but then it just returns an error

**Reply by kof** [reply]
I think you hit this bug https://github.com/webstudio-is/webstudio/issues/4250

Instance must be selected where generation needs to happen, we need to make sure it takes body by default.
